server/src/Q3Server/Properties/PublishProfiles/Q3MobileTest.pubxml

```diff
@@ -0,0 +1,28 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--
+This file is used by the publish/package process of your Web project. You can customize the behavior of this process
+by editing this MSBuild file. In order to learn more about this please visit http://go.microsoft.com/fwlink/?LinkID=208121. 
+-->
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <WebPublishMethod>MSDeploy</WebPublishMethod>
+    <ADUsesOwinOrOpenIdConnect>False</ADUsesOwinOrOpenIdConnect>
+    <LastUsedBuildConfiguration>Release</LastUsedBuildConfiguration>
+    <LastUsedPlatform>Any CPU</LastUsedPlatform>
+    <SiteUrlToLaunchAfterPublish />
+    <LaunchSiteAfterPublish>True</LaunchSiteAfterPublish>
+    <PrecompileBeforePublish>True</PrecompileBeforePublish>
+    <EnableUpdateable>True</EnableUpdateable>
+    <DebugSymbols>True</DebugSymbols>
+    <WDPMergeOption>DonotMerge</WDPMergeOption>
+    <ExcludeApp_Data>False</ExcludeApp_Data>
+    <MSDeployServiceURL>poolq3.zoo.lan</MSDeployServiceURL>
+    <DeployIisAppPath>Q3MobileTest</DeployIisAppPath>
+    <RemoteSitePhysicalPath />
+    <SkipExtraFilesOnServer>False</SkipExtraFilesOnServer>
+    <MSDeployPublishMethod>WMSVC</MSDeployPublishMethod>
+    <EnableMSDeployBackup>True</EnableMSDeployBackup>
+    <UserName>zoo\mai</UserName>
+    <_SavePWD>True</_SavePWD>
```

Comment:
It is useful to have this checked in - Gareth did the same for the main deployment profile.
This will make deployment much easier for the next person, although we probably should document this in Swiki as well. I'll do that.